examples/platforms/cc2650/radio.c

```diff
@@ -1840,7 +1842,24 @@ void cc2650RadioProcess(otInstance *aInstance)
         else
 #endif /* OPENTHREAD_ENABLE_DIAG */
         {
-            otPlatRadioTransmitDone(aInstance, &sTransmitFrame, sReceivedAckPendingBit, sTransmitError);
+            otRadioFrame ackFrame;
+            uint8_t psdu[IEEE802154_ACK_LENGTH];
+
+            ackFrame.mPsdu = psdu;
+            ackFrame.mLength = IEEE802154_ACK_LENGTH;
+            ackFrame.mPsdu[0] = IEEE802154_FRAME_TYPE_ACK;
+
+            if (sReceivedAckPendingBit)
+            {
+                ackFrame.mPsdu[0] |= IEEE802154_FRAME_PENDING;
+            }
+
+            ackFrame.mPsdu[1] = 0;
+            ackFrame.mPsdu[2] = sTransmitFrame.mPsdu[IEEE802154_DSN_OFFSET];
+
+            otRadioFrameFillCRC(&ackFrame);
```

Comment:
This should not be necessary.  OpenThread does not use the IEEE 802.15.4 FCS field.